docs/website/docs/command-reference/docs-mdx/init/devfile_from_any_registry_output.mdx

```diff
@@ -0,0 +1,15 @@
+```console
+$ odo init --devfile nodejs-react --name my-nr-app
+  __
+ /  \__     Initializing a new component
+ \__/  \
+ /  \__/    odo version: v3.4.0
```

Comment:
The test fails when I rebase the branch after bump to 3.5.0.

Could we find a way to include the build/VERSION file in the doc, and proccess the mdx file to resolve the include, before to compare it?

Reply:
Oh, you're right! Let me see what can be done.

Reply:
Discussed in standup, we will manually update the docs right now if they fail. For version, we can update GH action to update it with every release.